bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -36,16 +36,15 @@ <h1><span class="highlight">{{ self.page_title() }}</span></h1>
   {{ _('Mozilla newsgroups are sponsored by <a href="%s">Giganews Newsgroups</a>, to whom we are very grateful.')|format('https://www.giganews.com/') }}
   <a href="https://www.giganews.com/"><img src="{{ static('img/mozorg/about/giganews.png') }}" alt="" height="25" width="89" /></a>
   </p>
-
+  <a href="https://www.google.com/" ><img
+    src="{{ static('img/logos/google/logo_google.svg') }}"
+    width="75" height="32" alt="Google" /></a>
```

Comment:
The new image ratio is a little different, so the logo looks a little squished. 

![image](https://user-images.githubusercontent.com/400117/61145272-e85ae500-a4ce-11e9-8983-9293069689c4.png)


Setting `height="25"` seems to be about right.